# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../clearsilver/clearsilver-0.10.5-perl-build-options.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Add PERL_BUILD_OPTIONS variable

This can be handy to inject custom options for building the perl bindings.

--- clearsilver-0.10.5/Makefile.orig	2011-05-18 17:50:29.716002345 +0200
+++ clearsilver-0.10.5/Makefile	2011-05-18 17:52:42.772002343 +0200
@@ -29,7 +29,7 @@
 	@for mdir in $(SUBDIRS); do \
 	  if test -d $$mdir; then \
 	    if test -f $$mdir/Makefile.PL -a ! -f $$mdir/Makefile; then \
-	      cd $$mdir; $(PERL) Makefile.PL PREFIX=$(prefix); cd ..; \
+	      cd $$mdir; $(PERL) Makefile.PL PREFIX=$(prefix) $(PERL_BUILD_OPTIONS); cd ..; \
 	    fi; \
 	    $(MAKE) -C $$mdir PREFIX=$(prefix); \
 	  fi; \
